### PR TITLE
Rename the serverhtml function to just html, in line with static html

### PR DESCRIPTION
**Pull request by rictic**
This makes three types of html function: normal, static, and server-only.

We want this named `html` so that syntax highlighting works by default, and to be consistent with the static `html` function.

### CODE REVIEW

File: packages/labs/ssr/README.md [hunk 1]
```diff
@@ -178,16 +178,16 @@ that template can be loaded and hydrated on the client to apply the same data.
 
 ## Server-only templates
 
-You can also write templates that will only render on the server. These templates can be used for rendering full documents, including the doctype, and rendering into elements that Lit normally cannot, like `<title>`, `<textarea>`, `<template>`, and non-executing `<script>` tags like `<script type="text/json">`. They are also slightly more efficient than normal Lit templates, because the generated HTML doesn't need to include markers for updating.
+`@lit-labs/ssr` also exports an `html` function for defining server-only templates. You can also write templates that will only render on the server. These templates can be used for rendering full documents, including the doctype, and rendering into elements that Lit normally cannot, like `<title>`, `<textarea>`, `<template>`, and non-executing `<script>` tags like `<script type="text/json">`. They are also slightly more efficient than normal Lit templates, because the generated HTML doesn't need to include markers for updating.
```
Comment (AndrewJakubowicz) [inline packages/labs/ssr/README.md:181]: Nit: "function" -> "tag function"?

File: packages/labs/ssr/src/test/integration/tests/basic.ts [hunk 2]
```diff
@@ -42,7 +42,7 @@ import {
 
 import {anyHtml, SSRTest, SSRTestDescription} from './ssr-test.js';
 import {AsyncDirective} from 'lit/async-directive.js';
-import {serverhtml} from '../../../lib/server-template.js';
+import {html} from '../../../lib/server-template.js';
```
Comment (AndrewJakubowicz) [inline packages/labs/ssr/src/test/integration/tests/basic.ts:45]: On line 9 there is an import `html` from 'lit'. I think this should be `{html as serverhtml}`

File: packages/labs/ssr/src/test/test-files/render-test-module.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ import {repeat} from 'lit/directives/repeat.js';
 import {classMap} from 'lit/directives/class-map.js';
 import {LitElement, css, PropertyValues} from 'lit';
 import {property, customElement} from 'lit/decorators.js';
-import {serverhtml} from '../../lib/server-template.js';
+import {html} from '../../lib/server-template.js';
```
Comment (augustjk) [inline packages/labs/ssr/src/test/test-files/render-test-module.ts:12]: This one should also be `html as serverhtml` as it clashes with the regular `html`.

File: packages/labs/ssr/README.md (hunk 1 above)
Comment (rictic) [reply packages/labs/ssr/README.md]: Rephrased!

File: packages/labs/ssr/src/test/integration/tests/basic.ts (hunk 2 above)
Comment (rictic) [reply packages/labs/ssr/src/test/integration/tests/basic.ts]: Ah, yep. Fixed

File: packages/labs/ssr/src/test/test-files/render-test-module.ts (hunk 3 above)
Comment (rictic) [reply packages/labs/ssr/src/test/test-files/render-test-module.ts]: Ah, yep, fixed

**Review by augustjk — APPROVED** [review]
🎉

**Review by AndrewJakubowicz — APPROVED** [review]

**Pull request merged**